portals/bring-your-own-code-samples/react-sample/README.md

```diff
@@ -0,0 +1,123 @@
+# Power Pages - Bring Your Own Code - React Sample
+
+This is a react based sample application to demonstrate how a react website can be hosted on Power Pages and integrate with features like authentication, authorization, web apis etc.
+
+This is a modern web application for managing car sales, inventory, and customer information.
+
+## Features
+
+- Authentication using Microsoft Entra Id (see [AuthButton.tsx](src/components/AuthButton.tsx) for implementation)
+- Authorization using Power Pages web roles (see [Layout.tsx:45-49](src/components/Layout.tsx#L45-L49) for implementation)
+- Fetching data from external data sources through [virtual tables](https://learn.microsoft.com/en-us/power-pages/configure/virtual-tables) and web apis (see [SalesLeads.tsx](src/pages/SalesLeads.tsx#L30-L63) for implementation)
+
+## Prerequisites
+
+- Node.js (v14 or higher)
+- npm (v6 or higher)
+
+## Installation
+
+1. Clone the repository:
+
+    ```bash
+    git clone https://github.com/microsoft/PowerApps-Samples
+    cd portals/bring-your-own-code-samples\react-sample\
+    ```
+
+1. Install dependencies:
+
+    ```bash
+    npm install
+    ```
+
+## Development
+
+To start the development server:
+
+```bash
+npm run dev
+```
+
+The application will be available at `http://localhost:5173`
+
+## Building for Production
+
+To create a production build:
+
+```bash
+npm run build
+```
+
+The built files will be in the `dist` directory.
+
+## Running on Power Pages
+
+Below steps will help you run this app in Power Pages.
+
+### Setup
+
+1. Install [Microsoft Power Platform CLI](https://learn.microsoft.com/en-us/power-platform/developer/cli/introduction?tabs=windows#install-microsoft-power-platform-cli). (Version should be >= 1.43.6)
+1. Allow `*.js` files by removing it from `Blocked Attachments` in `Privacy + Security` settings for your environment from Power Pages Admin Center.
```

Comment:
Use this sentence:
```
Use the [System Settings General tab](https://learn.microsoft.com/power-platform/admin/system-settings-dialog-box-general-tab) under the **Set blocked file extensions for attachments** setting to control the file types to be blocked.  More information [Block files by extension](https://learn.microsoft.com/power-apps/developer/data-platform/files-images-overview?view=dataverse-latest&tabs=webapi#block-files-by-extension)
```

Reply:
Are the docs links correct? I don't see any System settings here.

![image](https://github.com/user-attachments/assets/90935307-f26a-4bbb-bc83-40a8dc6afabd)

Also, the blocked files by extension link is opening docs for controlling this through odata which is not required. Customers can easily do it from admin center.

We want customers to allow `*.js` files so it's better to write that explicitly.